src/v/config/configuration.cc

```diff
@@ -1561,6 +1561,34 @@ configuration::configuration()
       "cloud_storage_segment_size_target/2",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::nullopt)
+  , cloud_storage_max_download_throughput_per_shard(
+      *this,
+      "cloud_storage_max_download_throughput_per_shard",
+      "Max throughput used by tiered-storage per shard in bytes per second. "
+      "This value is an upper bound of the throughput available to the "
+      "tiered-storage subsystem. This parameter is intended to be used as a "
+      "safeguard and in tests when "
+      "we need to set precise throughput value independent of actual storage "
+      "media. "
+      "Please use 'cloud_storage_throughput_limit_percent' instead of this "
+      "parameter in the production environment.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1_GiB)
+  , cloud_storage_throughput_limit_percent(
+      *this,
+      "cloud_storage_throughput_limit_percent",
+      "Max throughput used by tiered-storage per node expressed as a "
+      "percentage of the disk bandwidth. If the server has several disks "
+      "Redpanda will take into account only the one which is used to store "
+      "tiered-storage cache. Note that even if the tiered-storage is allowed "
+      "to use full bandwidth of the disk (100%) it won't necessary use it in "
+      "full. The actual usage depend on your workload and the state of the "
+      "tiered-storage cache. This parameter is a safeguard that prevents "
+      "tiered-storage from using too many system resources and not a "
+      "performance tuning knob.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      50,
```

Comment:
How did you choose this value? Did it come from Abhijat's recent load testing?

Reply:
It's not really based on anything specific. We can probably experiment and find a sweet spot eventually but I think that this can be done as a followup.

Reply:
`50` feels a bit much to me, but let's fine tune later.